content/es/docs/reference/glossary/replica-set.md

```diff
@@ -0,0 +1,19 @@
+---
+title: ReplicaSet
+id: replica-set
+date: 2018-05-16
+full_link: /docs/concepts/workloads/controllers/replicaset/
+short_description: >
+  El ReplicaSet es la nueva generación del ReplicationController.
+
+aka: 
+tags:
+- fundamental
+- core-object
+- workload
+---
+ El ReplicaSet es la nueva generación del ReplicationController.
+
+<!--more--> 
+
+Un ReplicaSet, análogamente a un {{< glossary_tooltip text="ReplicationController" term_id="replication-controller" >}}, garantiza que un número establecido de réplicas de un pod estén corriendo en un momento dado. El ReplicaSet tiene soporte para selectores del tipo set-based, lo que permite el filtrado de claves por grupos de valores como por ejemplo todos los pods cuya etiqueta `environment` no sea `production` ni `qa`. Por otro lado, el ReplicationController solo soporta selectores equality-based, es decir, que solo puedes filtrar por valores exactos como por ejemplo, los pods que tengan la etiqueta `tier` con valor `frontend`-
```

Comment:
Sugiero utilizar *al igual que* que *análogamente*.

```suggestion
Un ReplicaSet, al igual que un {{< glossary_tooltip text="ReplicationController" term_id="replication-controller" >}}, garantiza que un número establecido de réplicas de un pod estén corriendo en un momento dado. El ReplicaSet tiene soporte para selectores del tipo set-based, lo que permite el filtrado de claves por grupos de valores como por ejemplo todos los pods cuya etiqueta `environment` no sea `production` ni `qa`. Por otro lado, el ReplicationController solo soporta selectores equality-based, es decir, que solo puedes filtrar por valores exactos como por ejemplo, los pods que tengan la etiqueta `tier` con valor `frontend`-
```

Reply:
No veo el problema de utilizar "análogamente". Es un sinónimo perfectamente válido. Entiendo que, la documentación, cuanto más fácil de entender mejor. Pero no creo "análogamente" sea ni ambiguo ni poco utilizado.   Just my opinion.

Reply:
Ambas propuestas son completamente válidas, como comenta @glo-pena, análogamente es muy común, a ver qué opina el autor.

Reply:
In my opinion, y pese a que "al igual" es 100% correcto, no veo suficientes motivos para cambiar "análogamente" por "al igual".
Así que, **lo dejaría como está.**